ngraph/core/src/op/min.cpp

```diff
@@ -22,7 +22,7 @@ namespace minop
     {
         out->set_shape(reduce(arg->get_shape(), axes, keep_dims));
         runtime::reference::min(
-            arg->get_data_ptr<ET>(), out->get_data_ptr<ET>(), arg->get_shape(), axes, keep_dims);
+            arg->get_data_ptr<ET>(), out->get_data_ptr<ET>(), arg->get_shape(), axes);
         return true;
     }
 
```

Comment:
Please update the ReduceMin::evaluate method below to reuse `get_normalized_axes_from_tensor` instead of   `get_reduction_axes()`

Reply:
Done